Sources/Clappr_iOS/Classes/Plugin/Container/PosterPlugin.swift

```diff
@@ -35,7 +33,7 @@ open class PosterPlugin: UIContainerPlugin {
     fileprivate typealias PosterUrl = String
     fileprivate func setPosterImage(with urlString: PosterUrl) {
         if let url = URL(string: urlString) {
-            poster.kf.setImage(with: url)
+            poster.download(url: url)
```

Comment:
What do you think about naming the method `display(url: URL)`? This is what the method does, right? It displays an image.